counsel.el

```diff
@@ -2424,6 +2424,11 @@ FZF-PROMPT, if non-nil, is passed as `ivy-read' prompt argument."
                         (message (cdr x)))
               :caller 'counsel-rpm)))
 
+(defcustom counsel-file-jump-args " * -type f -not -path '*\/.git*'"
+  "Arguments for the find-command when using counsel-file-jump."
```

Comment:
```
"String of arguments to `find-command' used by `counsel-file-jump'."
```

Reply:
Documentation string updates are still missing.  All symbols need to be quoted as shown in previous comment.